README.md

```diff
@@ -111,13 +111,15 @@ You can run the formatter manually by running `make format-yaml`.
 	cd repository-data/webfiles
 	mvn com.github.warmuuh:libsass-maven-plugin:0.2.8-libsass_3.4.4:watch
 
-**To provision and Run the local dev server without autoexport**
+**Run the local dev server (will throw a lot of Splunk errors)**
 
-	make serve.noexport
+	mvn clean verify && mvn -P cargo.run
 
-**To provision and Run the local dev server with autoexport (default)**
+**Run the local dev server with quiet Splunk**
+    
+    mvn -P cargo.run -D splunk.hec.name=localhost -D splunk.url=http://localhost -D splunk.token=
 
-	make serve
+**Both the CMS and the CMS console has their own AutoExport switch, which can be changed separately.**
```

Comment:
They are actually the same switch, if you turn on in one it will turn on in the other as well.